src/components/common/FilterGroup/index.tsx

```diff
@@ -148,9 +153,18 @@ export default function FilterGroup(props: FilterGroupProps): JSX.Element {
                 payloads={filters[f.name]?.children ?? []}
               />
             )}
+            {f.type === 'boolean' && (
```

Comment:
Let me know if you'd rather I call this type `'checkbox'` and the component `FilterCheckbox`

Reply:
`boolean` works for me